crawl-ref/source/state.h

```diff
@@ -30,6 +30,16 @@ struct god_act_state
     int      depth;
 };
 
+struct map_mode_info_t
+{
+public:
+    // If we are in map mode, absolute level pos of the cursor (position and branch id)
```

Comment:
I think you can leave off the "If" preface in these comments. Really anytime you have a valid `map_mode_info` its members are valid; the comment in `game_state` explaining that we should sometimes treat that particular `map_mode_info` as invalid is sufficient.